ArtemShchepets/src/main/java/week1/model/Bill.java

```diff
@@ -1,4 +1,4 @@
-package week1;
+package week1.model;
 
 public class Bill {
```

Comment:
Bill it is a model. All logic should be move to other class (BillController etc. Also implement interface)